src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -43,8 +86,35 @@
   {% set _css_class = _css_class ~ ' ecl-banner--' ~ _size %}
 {% endif %}
 
-{% if _centered %}
-  {% set _css_class = _css_class ~ ' ecl-banner--centered' %}
+{% if _font_size is not empty %}
+  {% set _css_class = _css_class ~ ' ecl-banner--font-' ~ _font_size %}
+{% endif %}
+
+{% set _color = '' %}
+{% if _font_color is not empty %}
+  {% set _color = _font_color %}
+{% endif %}
+
+{% if _box_background is not empty %}
```

Comment:
mmh..how would this be empty..? For a transparent box we expect a string "none" and since we set the default to light it cannot be empty.